helmfile.d/codefresh-re.yaml

```diff
@@ -0,0 +1,61 @@
+repositories:
+# Cloud Posse incubator repo of helm charts
+- name: "cloudposse-incubator"
+  url: "https://charts.cloudposse.com/incubator/"
+
+
+releases:
+
+#######################################################################################
+## Codefresh Runtime Environments                                                    ##
+#######################################################################################
+
+#
+# References:
+#   - https://codefresh-io.github.io/cli/runtime-environments/create-runtime-environments
+#   - https://github.com/cloudposse/charts/blob/master/incubator/monochart
+#   - https://kubernetes.io/docs/tasks/inject-data-application/define-command-argument-container
+#   - https://stackoverflow.com/questions/3790454/in-yaml-how-do-i-break-a-string-over-multiple-lines
+#
+- name: "codefresh-re"
+  namespace: "codefresh"
+  labels:
+    chart: "monochart"
+    component: "codefresh"
+    namespace: "codefresh"
+    vendor: "cloudposse"
+    default: "false"
+  chart: "cloudposse-incubator/monochart"
+  version: "0.8.0"
+  wait: true
+  values:
+    - fullnameOverride: codefresh-re
+      replicaCount: 1
+      image:
+        repository: codefresh/cli
```

Comment:
Use `build-harness` as the single point for commands.
It have tasks to install and run `codefresh/cli`

Reply:
@goruha I'm torn. Since this does one thing only (install codefresh runtime), I kind of like pinning to the official cli image. That makes updating it easy without needing to submit a PR against (a) packages (b) build-harness (c) helmfiles

Reply:
@osterman there is always a trade-off abstraction and repeatable.
If we would have the step in multiple repos (like we have with `slack notification` step) 
using build-harness would make code clear and simplify support.

But, as you said, it would make tailed PRs to made changes.

So from my side, the question is - how many repos per client would have this step?